proto/sequencerblockapis/astria/sequencerblock/v1alpha1/service.proto

```diff
@@ -4,6 +4,9 @@ package astria.sequencerblock.v1alpha1;
 
 import "astria/primitive/v1/types.proto";
 import "astria/sequencerblock/v1alpha1/block.proto";
+import "astria_vendored_sequencerblock/tendermint/abci/types.proto";
+import "astria_vendored_sequencerblock/tendermint/types/types.proto";
+import "astria_vendored_sequencerblock/tendermint/types/validator.proto";
```

Comment:
is there any reason these can't be named the same as they are in protocol `astria_vendored`? Does this create clashes?